README.md

```diff
@@ -66,6 +66,14 @@ But for some identifiers might have more. Check [their implementation](https://g
 
 For `URN`s, please check the [URN gem documentation](https://github.com/altmetric/urn) to see all the available options.
 
+### DOIs
+
+DOI's are notoriously tricky to pin down in a regex, since the actual regex could feasibly be `10\.{\d}+/\S+` in order to match all possible DOIs.
```

Comment:
This is not even supported with the current Regex and we probably should!
I'd like to test the current or any Regex candidate against our set of DOIs to check if we're able to detect/extract all of them (and which we're not), especially since the older regexes were more lax.